//[example_output
>example
Running 10 test cases...
test 1: 0
test 1: 1
test 1: 1
test 1: 2
test 1: 3
test 1: 5
test 1: 8
test 1: 13
test 1: 21
/Users/raffi/Personnel/boost_github/libs/test/doc/examples/example68.cpp:81: error: in "test_case_snippet_1": check fib_sample <= 13 has failed
Failure occurred in a following context:
    fib_sample = 21; index = 8; 
test 1: 34
/Users/raffi/Personnel/boost_github/libs/test/doc/examples/example68.cpp:81: error: in "test_case_snippet_1": check fib_sample <= 13 has failed
Failure occurred in a following context:
    fib_sample = 34; index = 9; 

*** 2 failures detected in test module "Master Test Suite"

//]